apps/root/src/pages/earn/home/frame/index.tsx

```diff
@@ -10,14 +10,15 @@ import useEarnService from '@hooks/earn/useEarnService';
 import useFilteredStrategies from '@hooks/earn/useFilteredStrategies';
 import { StrategiesTableVariants } from '@state/strategies-filters/reducer';
 import useAllStrategies from '@hooks/earn/useAllStrategies';
+import { strategyColumnConfigs } from '@pages/earn/components/strategies-table/components/columns';
 
 const variant = StrategiesTableVariants.ALL_STRATEGIES;
 
 const EarnFrame = () => {
   const dispatch = useAppDispatch();
   const earnService = useEarnService();
   const { strategies, hasFetchedAllStrategies } = useAllStrategies();
-  const filteredStrategies = useFilteredStrategies({ variant, strategies });
+  const filteredStrategies = useFilteredStrategies({ variant, strategies, columns: strategyColumnConfigs });
```

Comment:
Same comment could be applied here if we are not using filtered strategies for anything el se here